test/cmdlineTests.sh

```diff
@@ -35,6 +35,12 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-${REPO_ROOT}/build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+(( $# <= 1 )) || { printError "Too many arguments"; exit 1; }
+(( $# == 0 )) || [[ $1 == '--update' ]] || { printError "Invalid argument: '$1'"; exit 1; }
+
+AUTOUPDATE=""
```

Comment:
Initial value false instead of empty string?

Reply:
Changed.